project/plugins.sbt

```diff
@@ -12,7 +12,7 @@ addSbtPlugin("org.portable-scala"                % "sbt-scala-native-crossprojec
 addSbtPlugin("org.portable-scala"                % "sbt-scalajs-crossproject"      % "1.3.2")
 addSbtPlugin("org.scala-js"                      % "sbt-scalajs"                   % "1.16.0")
 addSbtPlugin("org.scalameta"                     % "sbt-mdoc"                      % "2.5.4")
-addSbtPlugin("org.scala-native"                  % "sbt-scala-native"              % "0.5.3")
+addSbtPlugin("org.scala-native"                  % "sbt-scala-native"              % "0.5.4")
```

Comment:
I tried updating this to 0.5.5 but there seems to be a bug or something has changed in the latest release and we get an OOM error during CI. I suspect that it's due to changes in the default memory allocations in Scala Native. I've [followed up with the Scala Native team on Discord](https://discord.com/channels/632150470000902164/635668881951686686/1274987097048219649) to see what might be the issue

Reply:
Seems indeed like a [Scala Native issue](https://github.com/scala-native/scala-native/issues/4032), unrelated to paralellism / changes in this PR 😮‍💨